### ISSUE TITLE
 Inquires about future work

### ISSUE THREAD

**Issue by exaids66** [xyongLu/SBCFormer#2]
Great job. Does the author have any plans to quantize the full-precision model? Additionally, are there any details to be aware of when deploying the full-precision model on edge devices?

**Comment by xyongLu** [comment]
Yes, quantization is also one part of my current study. Deployment on edge devices is the fundamental part of my research, but it's impossible to describe it in detail here.

**Comment by exaids66** [comment]
I completely understand that it's challenging to elaborate on technical specifics here. I’ve sent you an email inquiring about further details—if you're willing and have the time, I’d greatly appreciate your insights whenever convenient. Thank you!
